utilities/build_julia.sh

```diff
@@ -33,7 +33,12 @@ MFLAGS+=( "-j${JULIA_CPU_THREADS}")
 
 # Add a few default flags to our make flags:
 MFLAGS+=( "VERBOSE=1" )
-MFLAGS+=( "TAGGED_RELEASE_BANNER=Official https://julialang.org/ release" )
+if grep '1\.\(1[2-9]\|[2-9][0-9]\|[0-9][0-9][0-9][0-9]*\)\.' VERSION >/dev/null; then # if version >= 1.12
```

Comment:
We can make things a little less `awk`ward:

```suggestion
# NOTE: this breaks if you build v1.9 or lower with this
if awk "BEGIN {exit !($d1 >= 1.12)}"; then
```

We're not going to build v1.9 with this code, so the numeric comparison is fine.

Reply:
Not a fan of this change, but don't really care.

Reply:
```suggestion
# Taken from https://stackoverflow.com/a/4024263
verlte() {
    printf '%s\n' "$1" "$2" | sort -C -V
}
verlt() {
    ! verlte "$2" "$1"
}
if verlt "1.12" "$(cat VERSION)"; then
```